tests/tool_objectfs_testcase.php

```diff
@@ -252,4 +252,13 @@ protected function is_externally_readable_by_url($url) {
             return false;
         }
     }
+
+    protected function integration_client_supports_presigned_urls($client) {
+        $supportedlist = array();
+        $supportedlist[] = 'tool_objectfs\tests\test_s3_integration_client';
```

Comment:
Can you please explain why you check that the client supports presigned urls amnually and not using support_presigned_urls method?

Reply:
You're right, it's better to use `$externalclient->support_presigned_urls()` method. Re-written tests will be in further commit.